src/Components/SearchInput/SearchInput.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import SearchInput from './SearchInput'
+import { shallow } from 'enzyme';
+import { shallowToJson } from 'enzyme-to-json';
+
+
+
+describe('SearchInput', () => {
+    describe('props works correctly', () => {
+        it('theme is an empty string', () => {
+            const output = shallow(
+                <SearchInput placeholder="Search" theme=''/>
```

Comment:
Try to be consistent using " and '
placeholder="Search" theme=""
OR
placeholder='Search' theme=''

Reply:
Fixed.